scout/utils/link.py

```diff
@@ -156,8 +156,8 @@ def get_correct_ids(hgnc_symbol, alias_genes):
         hgnc_ids(iterable(int)): Hopefully only one but a symbol could map to several ids
     """
     hgnc_ids = set()
-    hgnc_symbol = hgnc_symbol.upper()
-    if hgnc_symbol in alias_genes:
+    # hgnc_symbol = hgnc_symbol.upper()
+    if hgnc_symbol in alias_genes or hgnc_symbol.upper() in alias_genes:
```

Comment:
To have a proper fix I think this should be:

```
if hgnc_symbol in alias_genes:
   hgnc_id_info = alias_genes[hgnc_symbol]
if hgnc_symbol.upper() in alias_genes:
   hgnc_id_info = alias_genes[hgnc_symbol.upper()]
```

Reply:
But also we must still be getting non-upper entries into the alias_genes map. I'll try to take a look whenever I get a moment..

Reply:
This link side, with OMIM, incomplete penetrance and exac info, seems good on its own, operating with upper case symbols for alias lookup. There is a complementary build aware system for nearly the same in the GeneHandler adapter class. I will try to set this up so we can use it directly instead for the alias collection as well, as the author apparently decided to use it for generating the alias map.

Reply:
Meh, I give up; too tired. Im afraid I will introduce bugs wrt build version this way. I used a version of your suggestion for now, and we can reconsider tomorrow. Im not sure it is completely build safe still either though, but follows a course of safe improvement. Let's test with like a panel with a hg38 only gene.